src/player.js

```diff
@@ -1659,6 +1686,11 @@ export default class Player extends FakeEventTarget {
           }
         });
       }
+      if (this.config.playback.playAdsWithMSE) {
+        this._eventManager.listen(this, AdEventType.AD_LOADED, this._detachMediaSource);
```

Comment:
move it to `AD_BREAK_START` as `AD_LOADED` could be while playback

Reply:
`AD_BREAK_START`  happened after ima took the control, it's too late

Reply:
@Yuvalke `AD_LOADED` may happen multiple times in an ad break, if you have multiple ads in one ad break. was it tested with such a use case?

Reply:
@OrenMe it tested, there is check for hls or shaka existence